recipe/meta.yaml

```diff
@@ -21,13 +21,21 @@ build:
   number: {{ build }}
   skip: true  # [win]
 
+# pin_compatible doesn't seem to find dependencies in output requirements
+# so list them here, even though they're unused.
+# Otherwise we get 'Could not get compatibility information for ucx package'
+requirements:
+  host:
+    - ucx  # [enable_cuda]
```

Comment:
This solved the render on 'openmpi', but now fails on 'openmpi-mpicc', which makes no sense because it doesn't even have `pin_compatible`.